compute/scheduler/node.go

```diff
@@ -6,15 +6,14 @@ import (
 
 	"github.com/ohsu-comp-bio/funnel/config"
 	"github.com/ohsu-comp-bio/funnel/logger"
-	pbs "github.com/ohsu-comp-bio/funnel/proto/scheduler"
 	"github.com/ohsu-comp-bio/funnel/util"
 	"github.com/ohsu-comp-bio/funnel/util/fsutil"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
-// NewNode returns a new Node instance
-func NewNode(ctx context.Context, conf config.Config, factory Worker, log *logger.Logger) (*Node, error) {
+// NewNodeInstance returns a new Node instance
```

Comment:
I don't like this. If the proto is getting in the way, it should be in a subpackage.

Reply:
I think we should either rename this type or the proto message type rather than use the same name for two different things.

Reply:
Ya, that is a bit confusing. How about calling the proto message something like `NodeInfo`?

Reply:
I am good with that.

Reply:
Sweet. Otherwise this all looks great. Thanks for doing this, it's a nice cleanup.